webpack.config.js

```diff
@@ -311,3 +322,12 @@ if (WEBPACK_BUNDLE !== "production") {
 
   config.devtool = "source-map";
 }
+
+function getLocalIPAddress(ipFamily) {
+  const networkInterfaces = os.networkInterfaces();
+  const interfaces = Object.keys(networkInterfaces).map(i => networkInterfaces[i]).reduce((p, q) => p.concat(q));
```

Comment:
nit:
1. It's easy to mistake `i` for an **index**, I assume you meant an **interface** here e.g. **eth0**.
2. can you fully name the variable `p` and `q`, my best guess would be `allAddresses` and `address` based on 
    > Each key on the returned object identifies a **network interface**. The associated value is an array of objects that each describe an assigned **network address**.

    this would help me visualize the result of [`os.networkInterfaces()`](https://nodejs.org/api/os.html#osnetworkinterfaces) without having to look at the documentation

Reply:
Adjusted for improved readability as well!